x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/IndexPrivilege.java

```diff
@@ -124,6 +126,7 @@ public final class IndexPrivilege extends Privilege {
     );
     private static final Automaton MANAGE_LEADER_INDEX_AUTOMATON = patterns(ForgetFollowerAction.NAME + "*");
     private static final Automaton MANAGE_ILM_AUTOMATON = patterns("indices:admin/ilm/*");
+    private static final Automaton MANAGE_DLM_AUTOMATON = patterns("indices:admin/dlm/*", "indices:admin/data_lifecycle/*");
```

Comment:
This shows we have a bit of an inconsistency in our action names. I'll bring the `explain` API into the `data_lifecycle` namespace (this comment is just a remark in general :) nothing about this PR)

Reply:
Opened https://github.com/elastic/elasticsearch/pull/95555 and actually brought all actions under the `dlm` namespace. 
If `data_lifecycle` is preferable we can go that way ?

Reply:
Thanks Keith, I merged https://github.com/elastic/elasticsearch/pull/95555 so let's update this to just use the `dlm` namespace